### PR TITLE
chore: New Crowdin translations to review and merge

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: lang/tr/texts/discover.html [hunk 1]
```diff
@@ -9,6 +9,7 @@ <h1>Open Food Facts</h1>
 		<h2 class="subheader">Özgür gıda ürünleri veritabanı.</h2>
 		<blockquote style="border-left:none;border-right: 1px solid #DDDDDD;">Tell me what you eat and I will tell you what you are. (Bana ne yediğini söyle, ben de sana kim olduğunu söyleyeyim.*)
 		(Bana ne yediğini söyle, ben de sana kim olduğunu söyleyeyim.*)
+		(Bana ne yediğini söyle, ben de sana kim olduğunu söyleyeyim.*)
```
Comment (hangy) [inline lang/tr/texts/discover.html:12]: 3rd time's a charm? 😄  Is this a bug in the translations or in Crowdin, which introduces the redundant translations?

File: lang/ach/texts/index-pro.html [hunk 2]
```diff
@@ -72,7 +72,7 @@ <h3>Compatible with EQUADIS, AGENA3000 (and soon more)</h3>
 </div>
 <div class="medium-4 columns">
 	<img src="https://world.openfoodfacts.org/images/producers/many-producers.png" alt="" style="height: 140px;">
-	<h3>Join the largest producers and distributors</h3>
+	<h3>10000</h3>
```
Comment (teolemon) [inline lang/ach/texts/index-pro.html:75]: NO

Comment (teolemon) [inline lang/ach/texts/index-pro.html:75]:
```suggestion
	<h3>Join the largest producers and distributors</h3>
```

File: lang/bg/texts/open-food-facts-mobile-app.html [hunk 3]
```diff
@@ -107,7 +107,7 @@ <h1 class="title-2 emphasized-title">
                     <span class="highlight-label blue rotate-right"></span>
                 </h1>
                 <p>Най-голямата в света отворена база данни за храни на една ръка разстояние!</p>
-                <a data-app-store="google" href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=bg" title="Open Food Facts в Google Play">
+                <a data-app-store="google" href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=bg" title="Отворете Food Facts в Google Play">
```
Comment (teolemon) [inline lang/bg/texts/open-food-facts-mobile-app.html:110]:
```suggestion
                <a data-app-store="google" href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=bg" title="Open Food Facts в Google Play">
```

File: lang/tr/texts/discover.html (hunk 1 above)
Comment (teolemon) [inline lang/tr/texts/discover.html:12]:
```suggestion
```

**Pull request merged**